core/src/main/java/eu/xenit/alfresco/AntiIdleScheduledJobExecuter.java

```diff
@@ -1,11 +1,38 @@
 package eu.xenit.alfresco;
 
+import static org.alfresco.service.namespace.QName.createQName;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.alfresco.model.ContentModel;
+import org.alfresco.repo.nodelocator.CompanyHomeNodeLocator;
+import org.alfresco.repo.nodelocator.NodeLocatorService;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
 import org.alfresco.service.ServiceRegistry;
+import org.alfresco.service.cmr.repository.ChildAssociationRef;
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.cmr.repository.NodeService;
+import org.alfresco.service.namespace.NamespaceService;
+import org.alfresco.service.namespace.QName;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.util.StringUtils;
 
 public class AntiIdleScheduledJobExecuter {
     private static final Logger LOG = LoggerFactory.getLogger(AntiIdleScheduledJobExecuter.class);
+    private static final String defaultLocation = "/Company Home/Data Dictionary/Anti Idle/Dummy Document";
+    private String[] folder = new String[]{"Data Dictionary", "Anti Idle"};
```

Comment:
Isn't this simply duplication ?

Reply:
Yes

I didn't deem it to be worth it to add a piece of code that automatically extracts the correct default segments from the defaultLocation String

Do you think that adding it would make this code easier to work with?

Reply:
Turns out to be not that much code after all
I'll plug it in